File: program65.c
 //Accept N numbers from user and return Smallest number between them.
#include<stdio.h>
#include<stdlib.h>

//Positive & Negative

int Minimum(int Arr[], int iLength)
{
    int iCnt = 0, iMin = Arr[0];

    for(iCnt = 0; iCnt < iLength; iCnt++)
    {
        if(iMin > Arr[iCnt] )
        {
           iMin = Arr[iCnt];
        }      
    }
    return iMin;
}

int main()
{
    int *ptr = NULL;
    int iSize = 0, iCnt = 0, iRet = 0;

    printf("Enter number of elements:\n");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));

    printf("Enter array elements:\n");
    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }

    iRet = Minimum(ptr, iSize);
    printf("Minimum number is: %d\n", iRet);
    free(ptr);

    return 0;
}

File: program65.java
//Accept N numbers from user and return Smallest number between them.

import java.lang.*;
import java.util.*;

class ArrayX
{
    private int Arr[];

    public ArrayX(int iNo)
    {
        Arr = new int[iNo];
    }

    public void Accept()
    {
        int iCnt = 0;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the values:");
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }
    public void Display()
    {
        int iCnt = 0;
        System.out.println("Elements in array are:");
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }
    }
    public int DisplaySmallest()
    {
        int iCnt = 0;
        int iMin = Arr[0];
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] < iMin)
            {
                iMin = Arr[iCnt];
            }
        }
        return iMin;
    }
}

class program65
{
    public static void main(String a[])
    {
        int iLength = 0, iRet = 0;
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter value for constructor:");

        iLength = sobj.nextInt();
        ArrayX obj = new ArrayX(iLength);

        obj.Accept();
        obj.Display();
        iRet = obj.DisplaySmallest();
        System.out.println("Smallest number is : "+ iRet);
    }
}